src/app/shared/interfaces/topic.interface.ts

```diff
@@ -34,15 +48,15 @@ export interface Map extends HasOpacity {
   minScale: number | null;
   /** True if unaccessible with current permissions. Not available in production environment. */
   permissionMissing?: boolean;
-  /** Timeslider Settings */
-  timeSliderConfiguration?: TimeSliderConfiguration;
   /** Filters Settings */
   filterConfigurations?: FilterConfiguration[];
   searchConfigurations?: SearchConfiguration[];
   /** Topic-specific notice for end-users */
   notice: string | null;
 }
 
+export type Map = BasicMap & TimeSliderSettings;
```

Comment:
@wes-EBP I know these are types, but I'm not sure we should extract those since they are directly tied to the `BasicMap`.

Reply:
Maybe this could be solved differently by using the _interface way_:
```typescript
timeSlider: undefined | {
  configuration: TimeSliderConfiguration;
  initialTimeSliderExtent: TimeExtent;
}
```

Reply:
Yes, but that would be another quite large refactor. Let's leave it and if it becomes an issue, we'll change it.